### PR TITLE
Move Assisted jobs from -2 to -elastic

### PR DESCRIPTION
**Pull request by jcpowermac**

### CODE REVIEW

**Comment by jcpowermac** [comment]
/pj-rehearse periodic-ci-openshift-assisted-test-infra-master-e2e-vsphere-assisted-periodic pull-ci-openshift-assisted-test-infra-master-e2e-vsphere-assisted-4-14

**Comment by jcpowermac** [comment]
/pj-rehearse periodic-ci-openshift-assisted-test-infra-master-e2e-vsphere-assisted-periodic pull-ci-openshift-assisted-test-infra-master-e2e-vsphere-assisted-4-14

**Comment by jcpowermac** [comment]
/pj-rehearse periodic-ci-openshift-assisted-test-infra-master-e2e-vsphere-assisted-periodic pull-ci-openshift-assisted-test-infra-master-e2e-vsphere-assisted-4-14

**Comment by jcpowermac** [comment]
/pj-rehearse periodic-ci-openshift-assisted-test-infra-master-e2e-vsphere-assisted-periodic

File: ci-operator/step-registry/ipi/conf/vsphere/lb/external/ipi-conf-vsphere-lb-external-commands.sh [hunk 1]
```diff
@@ -25,6 +25,7 @@ source "${SHARED_DIR}/govc.sh"
 unset SSL_CERT_FILE
 unset GOVC_TLS_CA_CERTS
 
+
```
Comment (vr4manta) [inline ci-operator/step-registry/ipi/conf/vsphere/lb/external/ipi-conf-vsphere-lb-external-commands.sh:28]: Is this line return ?  This is the only change in this file.

File: ci-operator/step-registry/cucushift/installer/check/vsphere/customized-resource/cucushift-installer-check-vsphere-customized-resource-commands.sh [hunk 2]
```diff
@@ -17,7 +17,7 @@ function get_node_names() {
    else
       mapfile -t  node_lists < <(oc get node --no-headers | awk '{print $1}')
    fi
-  
+
```
Comment (vr4manta) [inline ci-operator/step-registry/cucushift/installer/check/vsphere/customized-resource/cucushift-installer-check-vsphere-customized-resource-commands.sh:20]: Do these spaces need to be removed?  No other changes in this file.

**Comment by vr4manta** [comment]
/lgtm
other than the above nitpiks about whitespace changes, I'm good.  Feel free to fix any that you think should be changed.

**Comment by jianlinliu** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-vsphere-upi-encrypt-f28

**Comment by adriengentil** [comment]
Looks good to me, can you have a second eye @eliorerz ?

**Comment by jcpowermac** [comment]
@eliorerz please review

**Review by eliorerz — APPROVED** [review]

**Pull request merged**